components/automate-gateway/gateway/services.go

```diff
@@ -672,14 +672,25 @@ func (s *Server) ReportExportHandler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	writeContent(w, stream, query.Type)
+	writeContent(w, stream, query.Type, "NodeScans")
 }
 
-func writeContent(w http.ResponseWriter, stream reporting.ReportingService_ExportClient, queryType string) {
-	//when the type is json, we will be retrieving a json array of objects and since we will be getting them one at a
-	// time, we need to provide the '[' to open and the ']' to close (the close will happen on EOF, below)
-	if queryType == "json" {
-		_, err := w.Write([]byte("["))
+func writeContent(w http.ResponseWriter, stream reporting.ReportingService_ExportClient, queryType string, wrapper string) {
+	//we will be retrieving an array of objects and since we will be getting them one at a time, we need to provide the
+	//opening and closing elements to close for when we are dealing with json or xml (the close will happen on EOF, below)
+	//csv has no wrapping element so there is no element to prepend or append for csv type
+	var openBracket string
+	var closeBracket string
+	switch queryType {
+	case "json":
+		openBracket = "["
+		closeBracket = "]"
+	case "xml":
+		openBracket = fmt.Sprintf("<%s>\n", wrapper)
+		closeBracket = fmt.Sprintf("\n</%s>", wrapper)
+	}
+	if len(openBracket) == 0 {
```

Comment:
You seem to have the polarity of these guards backwards--should be `!=` here and on L702.
(Curious though, any reason `if openBracket != nil` would not work...? It is one less function call.)